tests/generate_plan_tests.rs

```diff
@@ -975,11 +971,7 @@ fn test_clojure_ring_app() -> Result<()> {
     );
     assert_eq!(
         plan.build.unwrap().cmds,
-        Some(vec![format!(
```

Comment:
Hmm, this has been formatted with `cargo fmt` is there is multiple flavors for?

Reply:
Hmm? Wdym?